scripts/index.js

```diff
@@ -0,0 +1,154 @@
+const state = {
+    temperature: 70,
+    sky: "sunny",
+    cityName: "Antioch, CA"
+}
+
+weather_info = {
+    cool: {
+        fontColor: "#57b3b5",
+        color: "#8cd1ec",
+        emojis: "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+        temp: [69,60]
+    },
+    warm: {
+        fontColor: "#d19419",
+        color: "FFE1B4",
+        emojis: "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+        temp: [79,70]
+    },
+    hot: {
+        fontColor: "#ec0c0c",
+        color: "#ec0c0c",
+        emojis: "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+        temp: [200, 80]
+    },
+    freezing: {
+        fontColor: "#7f96b5",
+        color: "#3f63c5",
+        emojis: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲",
+        temp: [59, -100]
+    }
+}
+
+sky_info = {
+    sunny: "https://images.unsplash.com/photo-1473496169904-658ba7c44d8a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1650&q=80",
+    cloudy: "https://images.unsplash.com/photo-1479688895406-3f032f15f0ef?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80",
+    rainy: "https://images.unsplash.com/photo-1610741083757-1ae88e1a17f7?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80",
+    snowy: "https://images.unsplash.com/photo-1518873890627-d4b177c06e51?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80"
+}
+
+const changeTempColor = () => {
+    const tempValue = document.querySelector("#tempValue");
+    const tempDisplay = document.querySelector("#tempDisplay");
+
+    if(state.temperature >= 80){
```

Comment:
If we structure the data used here similar to my suggestions, then rather than having hard-coded `if` conditions, we could iterate over the data to find the object that contains the current temperature. Then we could support an arbitrary number of temperature ranges!